integration/nwo/token/platform.go

```diff
@@ -197,12 +199,16 @@ func (p *Platform) PublicParametersDir() string {
 }
 
 func (p *Platform) PublicParametersFile(tms *topology2.TMS) string {
+	filename := fmt.Sprintf("%s_%s_%s_%s", tms.Network, tms.Channel, tms.Namespace, tms.Driver)
+	if len(tms.Alias) != 0 {
```

Comment:
What does this alias stand for?

Reply:
it is a way to distinguish two TMSs for the same triplet `network, channel, namespace`. 
In the update integration test, I need to generate different public parameters for the same triplet in order to perform the test. The alias allows me to distinguish them